plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExMemory.hs

```diff
@@ -276,3 +280,15 @@ instance ExMemoryUsage Data where
               sizeDataList (d:ds) = sizeData d + sizeDataList ds
               sizeDataPairs []           = 0
               sizeDataPairs ((d1,d2):ps) = sizeData d1 + sizeData d2 + sizeDataPairs ps
+
+instance ExMemoryUsage Crypto.BLS12_381.G1.Element where
+    memoryUsage _ = fromIntegral $ Crypto.BLS12_381.G1.memSizeBytes `div` 8
+    -- Should be 12
```

Comment:
Well it _is_ 12, but I thought it was better to get it from the library rather than hard-coding a constant.  I added the comment because it's hard to find out what the value is if you don't happen to know it already.